src/pages/Homepage.js

```diff
@@ -1,14 +1,31 @@
 import React, { useState } from "react";
+import cloud from '../images/cloud.png'
+// import DegreeToggle from "./DegreeToggle";
 
-const Homepage = () => {
+const Homepage = (props) => {
   const ApiKey = "3f5b56339f8fdbdc01e08aed47d34ac9";
-  const [weatherData, setWeatherData] = useState([{}]);
+  const [weatherData, setWeatherData] = useState([{}]); 
+  const [defaultLoaded, setDefaultLoaded] = useState(false);
   const [city, setCity] = useState("");
+  const [degreeType, setDegreeType] = useState("fahrenheit");
+
+  React.useEffect(() => {
+    if (!defaultLoaded) {
+      fetch(
+        `https://api.openweathermap.org/data/2.5/weather?q=Accra&units=imperial&APPID=${ApiKey}`
+      )
+        .then((response) => response.json())
+        .then((data) => {
+          setWeatherData(data);
+          setCity("");
+        })
+        .finally(() => setDefaultLoaded(true));
+    };
+  });
```

Comment:
This is neat but there is another way we could make this better and more performant. Add to the list of things we have to discuss. Advanced useEffects.